src/ZPublisher/tests/test_cookie.py

```diff
@@ -143,12 +143,12 @@ def test_domain(self):
         self.assertEqual(v, "fussball.example")
         # check ``bytes`` handling
         _, v = convertCookieParameter("domain",
-                                      "Fußball.example".encode("utf-8"))
```

Comment:
I'd rather keep the explicit encoding for clarity instead of relying on defaults.

Reply:
This must have been the choice of the pre commit hook. If I change it now, the next person will have the same problem.
Any way around that?

Reply:
This rewrite is caused by [pyupgrade](https://github.com/asottile/pyupgrade#encodeutf-8) and it is - unfortunately - not configurable. I think the only way to prevent this - while keeping `pyugrade` and `pre-commit` - would be to exclude the complete file via configuring the pre-commit hook in `pre-commit-config.yaml`.

Reply:
😒 Never mind then... (this is why I strongly dislike automated code changes using tools where someone else makes choices for you. Some of these choices are bad or not appropriate.)